app/grandchallenge/datatables/views.py

```diff
@@ -28,17 +28,27 @@ def get_context_data(self, *, object_list=None, **kwargs):
         )
         return context
 
-    def render_row(self, *, object_, page_context):
-        return render_to_string(
-            self.row_template, context={**page_context, "object": object_}
-        ).split("<split></split>")
+    def get_row_context(self, *, object_, page_context):
+        context = dict(**page_context)
+        context["object"] = object_
+        return context
+
+    def render_row(self, *, row_context):
+        return render_to_string(self.row_template, context=row_context).split(
+            "<split></split>"
+        )
 
     def render_rows(self, *, object_list):
-        page_context = self.get_context_data(object_list=object_list)
-        return [
-            self.render_row(object_=o, page_context=page_context)
-            for o in object_list
-        ]
+        page_context = self.get_context_data()
+        rows = []
+        for o in object_list:
+            row_context = self.get_row_context(
+                object_=o, page_context=page_context
+            )
+            rows.append(
+                self.render_row(row_context=row_context),
+            )
+        return rows
```

Comment:
Why was this file changed? The API now does not match its superclass.

Reply:
I originally saw the need to add a row context and manipulate that in subclasses.

However, with 'voortschrijdend inzicht' I see that I can work around that problem by referencing the algorithm via the job object itself.